app/code/Magento/Braintree/Controller/Paypal/Review.php

```diff
@@ -91,11 +91,14 @@ public function execute()
     }
 
     /**
-     * @param array $requestData
-     * @return boolean
+     * @param $requestData
+     * @return bool
      */
-    private function validateRequestData(array $requestData)
+    private function validateRequestData($requestData)
```

Comment:
Instead of changing this function, please fix a place where it is called

> does not get declared as an array as the requested post value returns a null, causing the validateRequestData($requestData)

Reply:
Not sure it's the best way to go. It's a method function and its purpose is to validate the request data. So, checking the type might be a part of the validation. It looks like an overhead to create an additional validation alongside with the method-validator.